x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/risk_score/risk_score_data_client.ts

```diff
@@ -331,4 +338,25 @@ export class RiskScoreDataClient {
       body: oldComponentTemplate.component_template,
     });
   }
+
+  public async reinstallTransform() {
```

Comment:
so is the implication here that the last 24h or alerts for users and hosts will be re-processed?

Reply:
@hop-dev
Not exactly. This is the latest transform; the origin datastream is `risk-score.risk-score-{space}-***`. From what I understand, it will reprocess all documents in the index.

Writing that down made me realise it could be super for users who installed the risk engine a long time ago. 
Should we add a time range filter to this transform? Theoretically, the engine runs every hour and regenerates the risk score for the last 24 hours of alerts. Should we add a two-hour lookback filter? 🤔

Reply:
That does sound reasonable, maybe we do 24h to be safe?

Reply:
Done! https://github.com/elastic/kibana/pull/203409/commits/775337aa3757b61df433d88afb6a4d9ad9f5b486